src/utils/api.js

```diff
@@ -0,0 +1,17 @@
+export const updatePassword = async (currentPassword, newPassword, confirmPassword) => {
+    const response = await fetch('http://localhost:3000/api/v1/users/update-password', {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorization': `Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjp7ImlkIjoiMjI1MTYwNzYtNWE3ZS00M2U3LWIxMzUtMDUzMzI1YzhmZDg0IiwiZW1haWwiOiJjYWx2aW5idWthcmFuaUBnbWFpbC5jb20iLCJyb2xlIjoiYnV5ZXIiLCJpc0FjdGl2ZSI6dHJ1ZSwiZXhwaXJlZCI6dHJ1ZX0sImlhdCI6MTY4NDE2MjYzM30.EeJV-B78xlXdm9--7XzEw7xdjg2F9lit716QIKPXM6A`
+      },
```

Comment:
why you hard coded this token?

Reply:
no that's the code before when i was testing lastly not the update